### PR TITLE
Allow disabling of color highlighting

### PR DESCRIPTION
**Pull request by gpanders**
It remains enabled by default.

### CODE REVIEW

File: vim/syntax/tmux.vim [hunk 1]
```diff
@@ -57,11 +57,13 @@ hi def link tmuxVariableExpansion Identifier
 
 " Make the foreground of colourXXX keywords match the color they represent.
 " Darker colors have their background set to white.
```
Comment (ericpruitt) [inline vim/syntax/tmux.vim:59]: Please update the comment with something like, "When tmux_syntax_colors is unspecified or set to a non-zero value, ..."

**Review by ericpruitt — CHANGES_REQUESTED** [review]
Please update README.md to document this feature along with the comment above the code block. I would also like the `\` on line 64 to remain in the first character column instead of being indented.

File: vim/syntax/tmux.vim (hunk 1 above)
Comment (gpanders) [reply vim/syntax/tmux.vim]: Done. I also moved the changes from `vim/syntax/tmux.vim` to `src/header.vim`.

**Comment by gpanders** [comment]
Done. I also moved the changes from `vim/syntax/tmux.vim` to `src/header.vim`.

**Comment by ericpruitt** [comment]
You need to regenerate tmux.vim by running `make`. Include the updated tmux.vim file in this pull request. You also still need to move the `\` to the first column on what is now line 65 header.vim.

**Comment by gpanders** [comment]
Done. Do you want me to squash this into a single commit?

**Review by ericpruitt — APPROVED** [review]

**Comment by ericpruitt** [comment]
That won't be necessary. Thanks for submitting this change.

**Pull request merged**